### PR TITLE
Feature/19197 api connection to connector AB#19197

### PR DESCRIPTION
**Pull request by ArtiomMatiom**

### CODE REVIEW

File: modules/azure/api_connectors/custom_connector/main.tf [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  template_content = file("./connection.json")
+  parameters_content = jsonencode({
+    "arm_connection_name" = {
+      value = var.connection_name
+    }
+    "location" = {
+      value = var.location
+    },
+    "api_connector_id" : {
```
Comment (tom-reinders) [inline modules/azure/api_connectors/custom_connector/main.tf:33]:
```suggestion
    "api_connector_id" = {
```

File: modules/azure/api_connectors/custom_connector/variables.tf [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+variable "connection_name" {
+  type        = string
+  description = "Name for the connection"
+}
+
+variable "connector_resource_group_name" {
+  type        = string
+  description = "Name for the connection"
```
Comment (tom-reinders) [inline modules/azure/api_connectors/custom_connector/variables.tf:13]: Fix description

File: modules/azure/api_connectors/custom_connector/variables.tf [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+variable "connector_name" {
+  type        = string
+  description = "Name for the connection"
```
Comment (tom-reinders) [inline modules/azure/api_connectors/custom_connector/variables.tf:18]: Fix description

**Review by tom-reinders — CHANGES_REQUESTED** [review]

File: modules/azure/api_connectors/custom_connector/variables.tf (hunk 2 above)
Comment (ArtiomMatiom) [reply modules/azure/api_connectors/custom_connector/variables.tf]: updated

File: modules/azure/api_connectors/custom_connector/variables.tf (hunk 3 above)
Comment (ArtiomMatiom) [reply modules/azure/api_connectors/custom_connector/variables.tf]: Updated

**Review by tom-reinders — APPROVED** [review]

**Pull request merged**